### DISCUSSION TITLE
A proposed nosology of inborn errors of metabolism

### DISCUSSION THREAD

**Discussion by pnrobinson** [monarch-initiative/mondo#4668] [Bigger project]
I think this article will be useful in building the hierarchy for inborn errors of metabolism
https://www.nature.com/articles/s41436-018-0022-8

**Comment by cmungall** [comment]
OK, we should start going through table S1. Perhaps we can request a more computable non-pdf version of this - would you be able to ask the authors if they have this? And if they want to follow progress?

I started looking at some of their groupings, it seems we are not 100% aligned

**Comment by nicolevasilevsky** [comment]
I converted that table to an Excel spreadsheet using Adobe Acrobat (I love the 'export to' feature!)
[TableS1_InbornErrorsOfMetabolism.xlsx](https://github.com/monarch-initiative/mondo/files/2128507/TableS1_InbornErrorsOfMetabolism.xlsx)

As a next step, would it be helpful if I mapped the terms in their list to MONDO terms, so we know what we have and what is missing?

**Comment by cmungall** [comment]
Why don't you start by mapping the grouping classes (and note any differences, since there these will not be exact). There is no need to manually map the specific diseases as this can be done by the OMIM gene IDs.

**Comment by cmungall** [comment]
Can you also investigate the diseases that have ?s in the OMIM column. We may already have some of these.

**Comment by nicolevasilevsky** [comment]
related to #1589

**Comment by nicolevasilevsky** [comment]
I created a google spreadsheet [here](https://docs.google.com/spreadsheets/d/1L1Rcz0wRbqpuhIa4ynID_u_eSRn0917W/edit#gid=1010116657)

**Comment by pnrobinson** [comment]
This would be a good place to get a lit of relevant diseases:

http://www.vademetab.org/

**Comment by nicolevasilevsky** [comment]
Nice, thanks, this is helpful!

**Comment by nicolevasilevsky** [comment]
Can we use some kind of tagging tool like Google OpenRefine to automatically map the parents and subclasses to Mondo IDs? (Name matching, then check that the OMIM IDs match)
@jiaola or @matentzn 

If this doesn't work, maybe Harshad can help.
